tools/config_updater.py

```diff
@@ -100,15 +105,39 @@
 parser_set_var.add_argument("var", help="Variable to set.")
 parser_set_var.add_argument("val", help="Value to set.")
 
+def AddUser(username, password, labels=[], token=None):
+  """Implementation of the add_user command."""
+  with aff4.FACTORY.Create("aff4:/users/%s" % username,
+                           "GRRUser", mode="rw", token=token) as fd:
+    # Note this accepts blank passwords as valid.
+    if not password:
+      password = getpass.getpass(
+          prompt="Please enter password for user '%s': " % username)
+    fd.SetPassword(password)
+
+    # Use sets to dedup input.
+    if labels is not []:
+      add_labels = set(labels)
+    else:
+      add_labels = None
+
+    if add_labels:
+      fd.AddLabels(*add_labels, owner="GRR")
+
+  print "Added user %s." % username
+
 
 def UpdateUser(username, password, add_labels=None, delete_labels=None,
                token=None):
   """Implementation of the update_user command."""
-  print "Updating user %s" % username
-  with aff4.FACTORY.Create("aff4:/users/%s" % username,
+
+  with aff4.FACTORY.Open("aff4:/users/%s" % username,
                            "GRRUser", mode="rw", token=token) as fd:
+
     # Note this accepts blank passwords as valid.
-    if password is not None:
+    if password:
```

Comment:
The logic around this one changed a bit since I moved the prompting for a password.  
1.  UpdateUser does not force a password update so you can add/remove labels w/o resetting passwords.
2.  UpdateUser uses --password as a flag to trigger a password prompt rather than --password="Foo" so you don't have to type your password on the command line.  

So if the password value is true it prompts and if false it skips.